nuxeo-common/src/main/java/org/nuxeo/common/utils/FileUtils.java

```diff
@@ -650,4 +648,15 @@ public static boolean areFilesContentEquals(String expected, String source) {
         }
     }
 
+    /**
+     * Replaces illegal characters (: \ / * ? | < >) in a filename with "_"
```

Comment:
Why those characters though? Why not others? For our use case the goal is to create a "safe" temporary relative file name. For that only `/` (unix) and `\` (windows) and ':' (mac, for the Finder) and `..` need to be stripped in my opinion.